grub2/ignition.cfg

```diff
@@ -0,0 +1,15 @@
+# Determine if this is a first boot and set the ${ignition_firstboot} variable
+# which is used in the kernel command line.
+set ignition_firstboot=""
+if [ -f "/ignition.firstboot" ]; then
+    # Default networking parameters to be used with ignition.
+    set ignition_network_kcmdline=''
+
+    # Source in the `ignition.firstboot` file which could override the
+    # above $ignition_network_kcmdline with static networking config.
+    # This override feature is also by coreos-installer to persist static
```

Comment:
```suggestion
    # This override feature is also used by coreos-installer to persist static
```

Reply:
(To be clear these are typos in the original source, but will fix anyways)